### DISCUSSION TITLE
Hiding get parameters from url when making a request using Inertia.visit - is it possible?

### DISCUSSION THREAD

**Discussion by armin-salihovic** [inertiajs/inertia#1269] [Help]
Hello, 

When using `Inertia.visit()` to make manual visits, is it possible to hide any get parameters?

The reason for this is comes from the usage of [spatie/laravel-query-builder](https://spatie.be/docs/laravel-query-builder/v5/introduction) package for Laravel. When making a get request with various filters, sorting, and pagination, I've noticed that the url becomes pretty cumbersome and unpleasant to look at.

For example, if I make an Inertia visit to `/users?filter[name]=john&filter[email]=gmail` is it possible to hide `?filter[name]=john&filter[email]=gmail` from the url visible in the browser?

One solution I can think of to achieve this is by making classic api calls with axios. However, I would like to use `Inertia.visit()` if possible as it keeps the code much simpler both on the frontend and the backend.

**Comment by fh32000** [comment]
any update ?
